bootstrap.d/app-editors.yml

```diff
@@ -1,4 +1,33 @@
 packages:
+  - name: mg
+    source:
+      subdir: 'ports'
+      git: 'https://github.com/troglobit/mg.git'
+      tag: 'v3.4'
+      version: '3.4'
+      tools_required:
+        - host-autoconf-v2.69
+        - host-automake-v1.15
+        - host-pkg-config
+      regenerate:
+        - args: ['./autogen.sh']
+    tools_required:
+      - host-pkg-config
+      - system-gcc
+    pkgs_required:
+      - ncurses
+    configure:
+      - args:
+        - '@THIS_SOURCE_DIR@/configure'
+        - '--host=x86_64-managarm'
+        - '--prefix=/usr'
```

Comment:
Right now, config files get installed into `/usr/etc`, passing `--sysconfdir=/etc` fixes this and places the config files in `/etc`

Reply:
Fixing the package
Will reopen when done.